src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -101,6 +101,9 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
   const isOnPlaya = pathname.toLowerCase() === venueInsideUrl(PLAYA_VENUE_ID);
 
+  // @debt remove when back button is removed from NavBar and/or admin v1 is no longer used
+  const isInAdmin = useRouteMatch("/admin/");
```

Comment:
[NB] Not using a constant in `settings.ts` as to contain  the `@debt` for this particular case in this particular line/file and not spread it further.

Reply:
reverted change in bbcc9c48252441cff5298c00280c10ac2c3eef35